packages/icrc-ledger-agent/src/lib.rs

```diff
@@ -262,11 +262,16 @@ impl Icrc1Agent {
         )?)
     }
 
-    pub async fn get_data_certificate(&self) -> Result<DataCertificate, Icrc1AgentError> {
-        Ok(Decode!(
-            &self.query("get_data_certificate", &Encode!()?).await?,
-            DataCertificate
-        )?)
+    pub async fn get_data_certificate(
+        &self,
+    ) -> Result<icrc_ledger_types::icrc3::blocks::ICRC3DataCertificate, Icrc1AgentError> {
```

Comment:
This type seems to be used in a few other places. Any reason for not adding a `use icrc_ledger_types::icrc3::blocks::ICRC3DataCertificate;` statement at the top of the file for better readability?

Reply:
Done